test/conformance/gateway_conformance_test.go

```diff
@@ -24,12 +24,25 @@ import (
 var skippedTestsForTraditionalRoutes = []string{
 	// core conformance
 	tests.HTTPRouteHeaderMatching.ShortName,
+	// There is an issue with KIC when processing this scenario.
+	// TODO: https://github.com/Kong/kubernetes-ingress-controller/issues/6136
+	tests.GRPCRouteListenerHostnameMatching.ShortName,
+	// tests.GRPCRouteHeaderMatching.ShortName and tests.GRPCExactMethodMatching.ShortName may
+	// have some conflicts, skipping either one will still pass normally.
+	tests.GRPCExactMethodMatching.ShortName,
+}
+
+var skippedTestsForExpressionRoutes = []string{
+	// When processingvthis scenario,
+	// the expression router of Kong currently needs to add `priority` to the route.
```

Comment:
My suggestion was to include as much information for future reader that it is clear what's the problem here.

This comment could be improved a bit to make it clearer like so (this also fixes the typo):

```suggestion
	// When processing this scenario, the Kong's expressions router requires `priority`
	// to be specified for routes.
	// We cannot provide that for routes that are part of the conformance suite.
```

WDYT?

Reply:
Great 👍 thank you